src/main/java/racing/core/domain/Car.java

```diff
@@ -1,21 +1,27 @@
 package racing.core.domain;
 
-import racing.core.dto.TrackInfo;
 import racing.core.exception.ErrorMessage;
 import racing.core.patterns.MoveStrategy;
 
 import java.util.Objects;
 
-public class Car implements Comparable<Car> {
+public final class Car implements Comparable<Car> {
 
     private static final int LENGTH_LIMIT = 5;
 
-    private String name;
-    private int position;
+    private final String name;
+    private final int position;
 
     public Car(String name) {
         validateName(name);
         this.name = name;
+        this.position = 0;
+    }
```

Comment:
생성자 내의 로직이 중복되네요 :) 아래처럼 고칠 수 있겠습니다.
```suggestion
    public Car(String name) {
        this(name, 0);
    }
```